Update TESTING.asciidoc (#66563)
elastic/elasticsearch: TESTING.asciidoc

==== Launching and debugging from an IDE

If you want to run Elasticsearch from your IDE, the `./gradlew run` task
supports a remote debugging option. Run the following from your terminal:

---------------------------------------------------------------------------
./gradlew run --debug-jvm
---------------------------------------------------------------------------

> Next, press the debug button icon within intelliJ (if using the pre configured debugger setting included in the elasticsearch repo) to attach to the process. This will enable the IDE to connect to the process and allow debug functionality.

Reviewer: ```suggestion
Next start the "Debug Elasticsearch" run configuration in IntelliJ. This will enable the IDE to connect to the process and allow debug functionality.
```

> The gradlew run --debug-jvm command will instruct all JVMs (including any that run cli tools such as creating the keyring or adding users) to suspend and initiate a debug connection on port incrementing from `5005`

Reviewer: ```suggestion
```
